frontend/src/component/changeRequest/ChangeRequestSidebar/EnvironmentChangeRequest/EnvironmentChangeRequest.tsx

```diff
@@ -36,113 +44,138 @@ export const EnvironmentChangeRequest: FC<{
+                            <>
+                                <SubmitChangeRequestButton
+                                    onClick={() =>
+                                        onReview(
+                                            environmentChangeRequest.id,
+                                            commentText
+                                        )
+                                    }
+                                    count={changesCount(
+                                        environmentChangeRequest
+                                    )}
+                                />
+
                                 <Button
-                                    sx={{ marginLeft: 2 }}
+                                    sx={{ ml: 2 }}
                                     variant="outlined"
-                                    onClick={() => {
-                                        onClose();
-                                        navigate(
-                                            `/projects/${environmentChangeRequest.project}/change-requests/${environmentChangeRequest.id}`
-                                        );
-                                    }}
+                                    onClick={() =>
+                                        onDiscard(environmentChangeRequest.id)
+                                    }
                                 >
-                                    View change request page
+                                    Discard changes
                                 </Button>
-                            </StyledFlexAlignCenterBox>
-                        </>
-                    }
-                />
+                            </>
+                        }
+                    />
+                    <ConditionallyRender
+                        condition={
+                            environmentChangeRequest.state === 'In review' ||
+                            environmentChangeRequest.state === 'Approved'
+                        }
+                        show={
+                            <>
+                                <StyledFlexAlignCenterBox>
+                                    <StyledSuccessIcon />
+                                    <Typography
+                                        color={theme.palette.success.dark}
+                                    >
+                                        Draft successfully sent to review
+                                    </Typography>
+                                    <Button
+                                        sx={{ marginLeft: 2 }}
+                                        variant="outlined"
+                                        onClick={() => {
+                                            onClose();
+                                            navigate(
+                                                `/projects/${environmentChangeRequest.project}/change-requests/${environmentChangeRequest.id}`
```

Comment:
I was thinking if we could centralize those links so that it's easier to maintain them